feature-libs/product-configurator/rulebased/components/price/configurator-price.component.spec.ts

```diff
@@ -97,7 +97,58 @@ describe('ConfiguratorPriceComponent', () => {
         '.cx-quantity-price'
       );
 
-      expect(component.price).toEqual('+ $10');
+      expect(component.price).toEqual('+$10');
+    });
+
+    it('should be defined: value price less than zero', () => {
```

Comment:
What is that supposed to mean? Please rephrase
(I understand that other tests in this file also have similar expressions)